src/main/java/teammates/sqllogic/core/FeedbackSessionsLogic.java

```diff
@@ -182,6 +187,48 @@ public FeedbackSession publishFeedbackSession(String feedbackSessionName, String
         return sessionToPublish;
     }
 
+    /**
+     * Gets the expected number of submissions for a feedback session.
+     */
+    public int getExpectedTotalSubmission(FeedbackSession fs) {
+        int expectedTotal = 0;
+
+        if (fqLogic.hasFeedbackQuestionsForStudents(fs.getFeedbackQuestions())) {
+            expectedTotal += usersLogic.getStudentsForCourse(fs.getCourse().getId()).size();
+        }
+
+        // Pre-flight check to ensure there are questions for instructors.
+        if (!fqLogic.hasFeedbackQuestionsForInstructors(fs.getFeedbackQuestions(), true)) {
+            return expectedTotal;
+        }
+
+        List<String> instructorEmails = usersLogic.getInstructorsForCourse(
+                fs.getCourse().getId()).stream().map(User::getEmail).collect(Collectors.toList());
```

Comment:
I don't think there's a need to map to instructorEmails. Would it be better to change this to a list of instructors instead?